src/main/java/com/minecolonies/coremod/network/messages/SaveScanMessage.java

```diff
@@ -1,15 +1,23 @@
 package com.minecolonies.coremod.network.messages;
 
+import com.minecolonies.blockout.Log;
 import com.minecolonies.coremod.util.ClientStructureWrapper;
 import io.netty.buffer.ByteBuf;
+import io.netty.buffer.ByteBufInputStream;
+import io.netty.handler.codec.EncoderException;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'io.netty.handler.codec.EncoderException'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUselessImportCheck)